engine/src/main/java/org/terasology/engine/rendering/world/RenderableWorldImpl.java

```diff
@@ -75,11 +77,11 @@
     private int statIgnoredPhases;
 
 
-    RenderableWorldImpl(Context context, GLBufferPool bufferPool, Camera playerCamera) {
```

Comment:
generating a VBO Id is a trivial cost and the expensive comes when you try to pass data to the buffer. the extra book keeping in maintaining a pool is probably more expensive then just disposing of the buffer.